src/mesh/coordinates.cxx

```diff
@@ -638,32 +638,34 @@ const Field3D Coordinates::Delp2(const Field3D &f) {
   int ncz = localmesh->LocalNz;
 
   // Allocate memory
-  auto ft = Matrix<dcomplex>(localmesh->LocalNx, ncz / 2 + 1);
-  auto delft = Matrix<dcomplex>(localmesh->LocalNx, ncz / 2 + 1);
+  Matrix<dcomplex> delft(localmesh->LocalNx, ncz / 2 + 1);
+  Matrix<dcomplex> ft(localmesh->LocalNx, ncz / 2 + 1);
+
+  static Laplacian *d2lap = Laplacian::defaultInstance();
+  d2lap->calcDelp2Coefs();
 
   // Loop over all y indices
   for (int jy = 0; jy < localmesh->LocalNy; jy++) {
 
     // Take forward FFT
 
+    #pragma omp parallel for
```

Comment:
If we declared `delft` and `ft` inside the `jy` loop could we move the omp parallelisation to the outer `jy` loop?